tools/Google.Cloud.SampleUtil/Google.Cloud.SampleUtil.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <PackageId>Google.Cloud.SampleUtil</PackageId>
```

Comment:
No need for this one. The description isn't really *necessary* either, but I think it's worth keeping.

Please add

```xml
<IsPackable>false</IsPackable>
```

Other than that, this looks fine.